vignettes/stan-help.Rmd

```diff
@@ -0,0 +1,423 @@
+#> 2: Log standard deviation for the parametric reference date delay
+#> 3:    Standard deviation of scaled pooled parametric mean effects
+#> 4:      Standard deviation of scaled pooled parametric sd effects
+#>             distribution mean sd
+#> 1:                Normal  1.0  1
+#> 2: Zero truncated normal  0.5  1
+#> 3: Zero truncated normal  0.0  1
+#> 4: Zero truncated normal  0.0  1
+#> 
+#> $inits
+#> function (data, priors) 
+#> {
+#>     priors <- enw_priors_as_data_list(priors)
+#>     fn <- function() {
+#>         init <- list(refp_mean_int = numeric(0), refp_sd_int = numeric(0), 
+#>             refp_mean_beta = numeric(0), refp_sd_beta = numeric(0), 
+#>             refp_mean_beta_sd = numeric(0), refp_sd_beta_sd = numeric(0))
+#>         if (data$model_refp > 0) {
+#>             init$refp_mean_int <- array(rnorm(1, priors$refp_mean_int[1], 
+#>                 priors$refp_mean_int[2]/10))
+#>         }
+#>         if (data$model_refp > 1) {
+#>             init$refp_sd_int <- array(abs(rnorm(1, priors$refp_sd_int[1], 
+#>                 priors$refp_sd_int[2]/10)))
+#>         }
+#>         init$refp_mean <- rep(init$refp_mean_int, data$refp_fnrow)
+#>         init$refp_sd <- rep(init$refp_sd_int, data$refp_fnrow)
+#>         if (data$refp_fncol > 0) {
+#>             init$refp_mean_beta <- array(rnorm(data$refp_fncol, 
+#>                 0, 0.01))
+#>             if (data$model_refp > 1) {
+#>                 init$refp_sd_beta <- array(rnorm(data$refp_fncol, 
+#>                   0, 0.01))
+#>             }
+#>         }
+#>         if (data$refp_rncol > 0) {
+#>             init$refp_mean_beta_sd <- array(abs(rnorm(data$refp_rncol, 
+#>                 priors$refp_mean_beta_sd_p[1], priors$refp_mean_beta_sd_p[2]/10)))
+#>             if (data$model_refp > 1) {
+#>                 init$refp_sd_beta_sd <- array(abs(rnorm(data$refp_rncol, 
+#>                   priors$refp_sd_beta_sd_p[1], priors$refp_sd_beta_sd_p[2]/10)))
+#>             }
+#>         }
+#>         return(init)
+#>     }
+#>     return(fn)
+#> }
+#> <bytecode: 0x55927f15dbb0>
+#> <environment: 0x559280451700>
+#> 
+
+```
+
+In the above example, if you have some reason to believe that the standard deviation of the means used could we smaller, you could reduce them by half to 0.5.
+
+```r
+
+new_priors <- default_priors$priors
+
+new_priors[ ,sd := 0.5]
+
+```
+
+You could then pass these new priors to the `epinowcast` function.
+
+```r
+epinowcast(pobs,
+  expectation = expectation_module,
+  fit = fit,
+  model = multithread_model,
+  priors = new_priors
+)
+```
+Similarly, you could take an empirical Bayes approach and use the posteriors of a [fitted model as described in another vignette](https://package.epinowcast.org/articles/germany-age-stratified-nowcasting.html#using-the-inflated-posterior-as-a-prior).
+
+
+## Exploring your data {.modelData}
+
+<!--
+#TODO Non-stationarity, longer time scales, multi-modality from known etiologies
```

Comment:
If this is a TODO for not this PR can we move it to a new issue - aiming to avoid any TODOs in the package  code/docs

Reply:
Removed and stowed [here](https://gist.github.com/medewitt/caf2af7095b2d851a0d9de736b489987
 for the moment until I put these notes into a proper issue.